### PR TITLE
Add new Firestore events supported by Eventarc.

### PR DESCRIPTION
**Pull request by minyu19**

### CODE REVIEW

**Comment by meteatamel** [comment]
@minyu19 Waiting for your go-ahead before merging.

**Comment by minyu19** [comment]
Thank you, Mete. Looks like it will take some more time. I will let you know when it is ready.

**Review by meteatamel — APPROVED** [review]

**Pull request merged**
